core/trino-main/src/main/java/io/trino/connector/system/MaterializedViewDto.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.connector.system;
+
+import io.trino.metadata.QualifiedObjectName;
+import io.trino.spi.connector.ConnectorMaterializedViewDefinition;
+import io.trino.spi.connector.MaterializedViewFreshness;
+
+public class MaterializedViewDto
```

Comment:
MaterializedViewDto -> MaterializedViewInfo

Reply:
How about `MaterializedViewHandle`? There are a few other entities named `SystemTransactionHandle`, `SystemTableHandle` for example

Reply:
*Handle classes typically implement Connector*Handle interface, calling this MaterializedViewHandle could be confusing, let's call it MaterializedViewInfo or something similar